Shared/Extensions/ViewExtensions/Modifiers/ScrollViewOffsetModifier.swift

```diff
@@ -19,7 +19,17 @@ struct ScrollViewOffsetModifier: ViewModifier {
     }
 
     func body(content: Content) -> some View {
-        content.introspect(.scrollView, on: .iOS(.v15), .iOS(.v16), .iOS(.v17), .tvOS(.v15), .tvOS(.v16), .tvOS(.v17)) { scrollView in
+        content.introspect(
+            .scrollView,
+            on: .iOS(.v15),
+            .iOS(.v16),
+            .iOS(.v17),
+            .iOS(.v18),
+            .tvOS(.v15),
```

Comment:
Since tvOS 17 is now minimum, should probably remove the references to earlier versions.